Sources/Clocks/Internal/Yield.swift

```diff
@@ -1,8 +1,15 @@
+import Foundation
+
 extension Task where Success == Failure, Failure == Never {
   // NB: We would love if this was not necessary, but due to a lack of async testing tools in Swift
   //     we're not sure if there is an alternative. See this forum post for more information:
   //     https://forums.swift.org/t/reliably-testing-code-that-adopts-swift-concurrency/57304
-  static func megaYield(count: Int = 10) async {
+  static func megaYield(
+    count: Int = max(
+      ProcessInfo.processInfo.environment["TASK_MEGA_YIELD_COUNT"].flatMap(Int.init) ?? 20,
+      20
+    )
```

Comment:
Added a `max(n, 20)` to avoid attack vectors where `1...-1` gets evaluated.

I suppose `TASK_MEGA_YIELD_COUNT = 9223372036854775807` could also be problematic, so maybe we also have a `min` limit?